### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hippogriffic**
@incuna/frontend please review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,11 +35,25 @@ Fallbacks for older browsers are the responsibility of the developer adding Hove
...
-2. Add the option to introduce arguments to mixins to make them more customisable
-3. Create some really great page transitions and other effects that will pair nicely with Angular JS's functionality.
-4. And more...
+This project uses `grunt` to compile sass and templates. 
+
+To set up the project locally:
+* Run `npm install`
+* Run `bower install`
+* Run `grunt`
+* Open `index.html` in your browser
```
Comment (Minglee01) [inline README.md:47]: i dont think this step is needed to actually set up the project.

File: README.md [hunk 2]
```diff
@@ -35,11 +35,25 @@ Fallbacks for older browsers are the responsibility of the developer adding Hove
...
-1. Remove all trace of icon-fonts. If you would like to learn more about why icon-fonts are bad you can watch [this video](https://www.youtube.com/watch?v=9xXBYcWgCHA) by our excellent contributor @ninjanails.
-2. Add the option to introduce arguments to mixins to make them more customisable
-3. Create some really great page transitions and other effects that will pair nicely with Angular JS's functionality.
-4. And more...
+This project uses `grunt` to compile sass and templates. 
+
+To set up the project locally:
+* Run `npm install`
+* Run `bower install`
+* Run `grunt`
```
Comment (Minglee01) [inline README.md:46]: i dont think this step is needed to actually set up the project. running the project then yes.

File: README.md [hunk 3]
```diff
@@ -35,11 +35,25 @@ Fallbacks for older browsers are the responsibility of the developer adding Hove
...
 
-1. Remove all trace of icon-fonts. If you would like to learn more about why icon-fonts are bad you can watch [this video](https://www.youtube.com/watch?v=9xXBYcWgCHA) by our excellent contributor @ninjanails.
-2. Add the option to introduce arguments to mixins to make them more customisable
-3. Create some really great page transitions and other effects that will pair nicely with Angular JS's functionality.
-4. And more...
+This project uses `grunt` to compile sass and templates. 
+
+To set up the project locally:
+* Run `npm install`
+* Run `bower install`
```
Comment (Minglee01) [inline README.md:45]: You could make a Makefile and combine these into a `make install` cmd

File: README.md (hunk 1 above)
Comment (henrahmagix) [inline README.md:47]: But is useful anyway

File: README.md (hunk 2 above)
Comment (henrahmagix) [inline README.md:46]: Yes, I think they should be separate steps:

``` md
## Install
- stuff

## Dev
- stuff
```

File: README.md (hunk 3 above)
Comment (henrahmagix) [inline README.md:45]: I don't think that's necessary. It's most useful for large projects that have required install steps that aren't obvious based on the files: `package.json` and `bower.json` both tell the dev to `npm install` and `bower install`, respectively.

Comment (Minglee01) [inline README.md:45]: @henrahmagix  note the `could` its a suggestion and not a you should do this.

Comment (henrahmagix) [inline README.md:45]: I know that, and it's a worthy suggestion =)

**Comment by hippogriffic** [comment]
ok to merge?

**Pull request merged**